lazyscribe/artifacts/__init__.py

```diff
@@ -23,11 +24,30 @@ def _get_handler(alias: str) -> Type[Artifact]:
         The artifact handler class object. This object will need to be constructed
         using :py:meth:`lazyscribe.artifacts.Artifact.construct`.
     """
-    for obj in Artifact.__subclasses__():
-        if obj.alias == alias:
-            out = obj
```

Comment:
Do you think we should get rid of this pattern? I'm wondering whether it's helpful for users that are experimenting outside of the packaging context.